app/main.py

```diff
@@ -14,4 +15,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    return len([i.lower() for i in phrase if letter.lower() == i.lower()])
```

Comment:
You could consider using the functional approach with chaining here as well, but this implementation is possible.